lib/api/src/js/externals/table.rs

```diff
@@ -116,4 +117,8 @@ impl Table {
     pub fn is_from_store(&self, _store: &impl AsStoreRef) -> bool {
         true
     }
+
+    pub fn store_id(&self) -> StoreId {
+        StoreId::unknown()
+    }
```

Comment:
Why this? Store's should be consistent per thread @john-sharratt

Reply:
It can be deleted - it was added because of the improved error messages which I have subsequently removed again - ill take it out

Reply:
deleted